scene/gui/color_picker.cpp

```diff
@@ -427,8 +427,13 @@ void ColorPicker::_html_submitted(const String &p_html) {
 		return;
 	}
 
+	Color new_color = Color::html(p_html);
+	if (color == new_color) {
+		return;
+	}
+	color = new_color;
+
 	float last_alpha = color.a;
```

Comment:
You broke this logic.

Reply:
I do not understand, I checked again, it seems to work fine. Is it the alpha value?

Reply:
Well to me the code is relatively self explanatory:
- `last_alpha` saves the previous alpha value
- `color` gets assigned a new color, which may have a different alpha value
- if `is_editing_alpha()` is disabled (i.e. the user wants to keep the original alpha value), the original alpha value is restored

After your change, this code is this:
```cpp
float last_alpha = color.a;
if (!is_editor_alpha()) {
	color.a = last_alpha;
}
```

Tautological, isn't it?

Reply:
Thank you for your eye, I have addressed it fully.